### ISSUE TITLE
Access to the path is denied error (not resolved by restarting Flagrum) after reinstalling a deleted mod

### ISSUE THREAD

**Issue by MrsFlowerpot** [Kizari/Flagrum#87]
I installed Rinual's Mod Menu and two subsequent updates.  I disabled the mod and found three copies of the mod in my mod manager.  I didn't know which version was the latest one so I deleted them.  When I tried to reinstall the latest version I got an error message - Access to the path is denied.  The application will now close.  I clicked OK and a second identical error message popped up.  I clicked OK again and Flagrum shut down.

I reopened Flagrum and found a copy of the mod in disabled mods.  I tried to enable it but would not work.  I deleted it, closed Flagrum, rebooted my computer and tried again but got the same error.

Please could you look into this?  I have attached crash.txt and log.txt files.
   
[crash.txt](https://github.com/Kizari/Flagrum/files/12541279/crash.txt)
[Log.txt](https://github.com/Kizari/Flagrum/files/12541281/Log.txt)

**Comment by Kizari** [comment]
This is a really weird bug that happens rarely and we have no idea why. Windows just decides that some of the Flagrum-created archives are no longer allowed to be overwritten by Flagrum for goodness knows what reason and it results in this issue. The only fix we've found for it was to delete the Flagrum-created archives and try again.

The only way we've found to resolve the issue is to delete the Flagrum-created archives for the affected mod manually, but that can be quite difficult as there are generally quite a few and they can be scattered throughout various folders. If you don't want to deal with that headache, the best approach is to disable all your mods (use the force disable feature that pops up if any of them are stuck on), then uninstall FFXV via Steam, go to the FFXV directory and delete any residual files as Steam won't clear up any of the Flagrum-created archives, then reinstall via Steam. My saves and settings were all preserved when I did this, but I do have Steam sync on, so not sure if that's why or if uninstalling just doesn't remove that stuff.
